src/misc/sw-template.js

```diff
@@ -299,7 +299,7 @@ function WebpackServiceWorker(params) {
 }
 
 function addAllNormalized(cache, requests) {
-  return Promise.all(requests.map(fetch)).then((responses) => {
+  return Promise.all(requests.map((request) => fetch(request))).then((responses) => {
```

Comment:
Yeah, I see. It cases errors. I think I am going merge this since I use `v3` on production and failed SW is a bad thing there..